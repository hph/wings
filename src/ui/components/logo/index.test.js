import React from 'react';
import renderer from 'react-test-renderer';

import Logo from './index';

test('Logo renders an SVG logo', () => {
  expect(renderer.create(<Logo />).toJSON()).toMatchSnapshot();
});
