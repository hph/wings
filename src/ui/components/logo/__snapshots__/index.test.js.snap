// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders an SVG logo 1`] = `
<div
  className="root"
  dangerouslySetInnerHTML={
    Object {
      "__html": "W.svg",
    }
  }
/>
`;
